docs/messaging/notifications.mdx

```diff
@@ -248,7 +248,7 @@ FirebaseMessaging.onMessage.listen((RemoteMessage message) {
           android: AndroidNotificationDetails(
             channel.id,
             channel.name,
-            channel.description,
+            description: channel.description,
```

Comment:
@IdrisAdeyemi01,  are you sure this isn't `channelDescription` for `AndroidNotificationDetails`?

Reply:
No isn't.

Reply:
No isn't.

Reply:
Is the example in the documentation out of date?
<img width="592" alt="Screenshot 2021-12-30 at 14 15 19" src="https://user-images.githubusercontent.com/16018629/147759725-92a7bae1-2d23-4b0e-8a82-a9d61f86080b.png">

Reply:
Yes it is

Reply:
Ok, I see you're correct [here](https://github.com/MaikuB/flutter_local_notifications/blob/master/flutter_local_notifications/lib/src/platform_specifics/android/notification_details.dart#L104) 👍 . Do you mind also updating the example app in the messaging package to use the latest local notifications package (i.e. version `9.2.0`)? [This will have to be updated](https://github.com/FirebaseExtended/flutterfire/blob/master/packages/firebase_messaging/firebase_messaging/example/lib/main.dart#L151). But there might be other breaking changes once you update to the latest version. Thanks 😄 !

Reply:
I have made the update and also made a push.

Reply:
You haven't updated the example app [here](https://github.com/FirebaseExtended/flutterfire/blob/master/packages/firebase_messaging/firebase_messaging/example/lib/main.dart#L151)